framework/src/modules/chain/transport/broadcaster.js

```diff
@@ -14,8 +14,8 @@
 
 'use strict';
 
-const _ = require('lodash');
-const jobsQueue = require('../utils/jobs_queue');
+const ENDPOINT_BROADCAST_TRANSACTIONS = 'postTransactionsAnnouncement';
```

Comment:
If you introduce this, you should make it in more places and tests as well. Chain file is using the event and transport spec.

Reply:
this constant is not exposed, it is just to make it clear in this file

Reply:
You could expose it and reuse it.

Reply:
Prefer not to expose this since this will be endpoint definition, and I think it make sense to keep it as it shouldn't be updated by mistake

Reply:
Then, why do you create a constant?? It doesn't make sense to create one here and not in the rest...